### ISSUE TITLE
public mails ?

### ISSUE THREAD

**Issue by ChoOo7** [piyushchauhan2011/Online-Treasure-Hunt#1]
Is normal that
https://github.com/piyushchauhan2011/Online-Treasure-Hunt/blob/master/techvibes_treasurehunt.sql
is public ? there is some emails

**Comment by piyushchauhan2011** [comment]
Yeah, I made some mistakes in the past that I wish I could take back, but what's done is done
